packages/html/src/drawer/templates/drawer-normal.tsx

```diff
@@ -0,0 +1,27 @@
+import { Drawer, DrawerContent, DrawerItem, DrawerItemSeparator } from "..";
+
+export const DrawerNormal = (props) => (
+    <Drawer expanded
```

Comment:
In my opinion DrawerNormal should not be expanded. I would suggest to add an additional template for expanded drawer